react/index.js

```diff
@@ -21,6 +21,7 @@ export { default as ListItemText } from './ListItemText'
 export { default as SelectionBar } from './SelectionBar'
 export { default as IntentIframe } from './IntentIframe'
 export { default as IntentOpener } from './IntentOpener'
```

Comment:
If it's deprecated, we can remove it from here, it is easy enough for an applicatin to replace the index export to `import IntentOpener from 'cozy-ui/dist/IntentOpener` if necessary.